pkg/devfile/adapters/interface.go

```diff
@@ -1,5 +1,5 @@
 package adapters
 
 type PlatformAdapter interface {
-	Push(path string, ignoredFiles []string, forceBuild bool, globExps []string) error
+	Push(path string, ignoredFiles []string, forceBuild bool, show bool, globExps []string, devfileBuildCmd, devfileRunCmd string) error
```

Comment:
As Rajiv mentioned higher up, this function and the other push function should be modified to accept a struct.

See my `odo watch` PR for how I did it: https://github.com/openshift/odo/blob/a727c2d31fe402a73f9beab046d72366a11123f7/pkg/devfile/adapters/common/types.go#L27

Reply:
thx, adopted `PushParameters` from the Watch PR, will rebase on other changes when it is merged